addon/mixins/list-input.js

```diff
@@ -0,0 +1,71 @@
+// import _ from 'lodash'
+import Ember from 'ember'
+// import computed, {readOnly} from 'ember-computed-decorators'
+
+import * as utils from '../components/utils'
+
+export default Ember.Mixin.create({
+  init () {
+    this._super(...arguments)
```

Comment:
@cstolli is correct here which is probably an argument against using a mixin over inheritance.